include/robot_localization/ros_filter.h

```diff
@@ -245,11 +248,13 @@ template<class T> class RosFilter
     //! @param[in] msg - The ROS stamped pose with covariance message to take in
     //! @param[in] callbackData - Relevant static callback data
     //! @param[in] targetFrame - The target frame_id into which to transform the data
+    //! @param[in] childFrame - The source frame_id from which to transform the data
```

Comment:
OK, I see your point. Please first read the comment in your feedback below.

....

So, what do you think about `poseSourceFrame` (in case you agree to applying the new behavior to the pose without `relative` and `differential` as well)?

Reply:
Also, renaming this would be great, thanks. Your name sounds fine.

Reply:
Done.